src/components/BadgeOverlay/styled/container.js

```diff
@@ -0,0 +1,19 @@
+import styled from 'styled-components';
+import attchThemeAttrs from '../../../styles/helpers/attachThemeAttrs';
+
+const StyledContainer = attchThemeAttrs(styled.span)`
+    position: relative;
+    display: inline-flex;
+    vertical-align: middle;
+    flex-shrink: 0;
+    width: 40;
```

Comment:
why do we need to add `width` and `height` to the container? I think this should have the size of the container.